cbt_export_changes.py

```diff
@@ -74,13 +73,17 @@ def save_changed_blocks(changed_blocks, output_file):
             out.write(b)
 
 
-def download_changed_blocks(bitmap, uri, changed_blocks_output_path,
+def download_changed_blocks(bitmap, get_nbd_info, changed_blocks_output_path,

```

Comment:
get_nbd_info=session.xenapi.VDI.get_nbd_info(new_snapshot_ref)[0] in this case

Two asks:
- Please add a variable to clarify what session.xenapi.VDI.get_nbd_info(new_snapshot_ref)[0] is.
- Don't call it by the function name get_nbd_info - this confused me.

Reply:
Sure